test/run_test.py

```diff
@@ -390,17 +389,16 @@ def get_test_time_reports_from_S3() -> List[Dict[str, Any]]:
         encoding="ascii").splitlines()
 
     stripped_job = get_stripped_CI_job()
-    bucket = get_S3_bucket_readonly('ossci-metrics')
     reports = []
     commit_index = 0
     while len(reports) == 0 and commit_index < len(nightly_commits):
         nightly_commit = nightly_commits[commit_index]
         print(f'Grabbing reports from nightly commit: {nightly_commit}')
-        summaries = bucket.objects.filter(Prefix=f"test_time/{nightly_commit}/{stripped_job}")
-        for summary in summaries:
-            binary = summary.get()["Body"].read()
-            string = bz2.decompress(binary).decode("utf-8")
-            reports.append(json.loads(string))
+        summaries = get_test_stats_summaries_for_job(sha=nightly_commit, job_prefix=stripped_job)
+        for _, summary in summaries.items():
+            reports.append(summary[0])
+            if len(summary) > 1:
+                print(f'Warning: multiple summary object found for {nightly_commit}/{stripped_job}')
```

Comment:
```suggestion
for job_name, summary in summaries.items():
            reports.append(summary[0])
            if len(summary) > 1:
                print(f'Warning: multiple summary object found for {nightly_commit}/{job_name}')
```
This would be more accurate right? Whereas stripped_job would be something like pytorch_linux_xenial_cuda10.2, job_name could be pytorch_linux_xenial_cuda10.2_test1 and pytorch_linux_xenial_cuda10.2_test2, right?